File: LeetCode-Programs/Find all duplicates in an Array.cpp
class Solution {
public:
    vector<int> findDuplicates(vector<int>& nums) {
     vector<int> v;
     for(int i=0;i<nums.size();i++)
     {
        int n=abs(nums[i]); 
        if(nums[n-1]<0) 
        v.push_back(n); 
        nums[n-1]=-nums[n-1];
        }
    return v;
}
};

File: LeetCode-Programs/Find all duplicates in an Array.py
class Solution:
    def findDuplicates(self, nums: List[int]) -> List[int]:
        nums.sort()
        num=[]
        s=len(nums)
        for i in range(1,s):
                if nums[i]==nums[i-1]:
                    num.append(nums[i])
        return num
